README.md

```diff
@@ -1,25 +1,40 @@
 # CloudFormation Information
 
-## What is it, and why?
-A script that scrapes all cloudformation stacks in an account and produces a report listing:
-  - The unique AWS resource types being used
-  - If the stack has been defined with [`@guardian/cdk`](https://github.com/guardian/cdk)
+A command line tool to audit Cloudformation stacks for a given set of AWS
+accounts.
 
-This is to help understand which stacks should be migrated next.
+The motivation is to assist with @guardian/cdk migrations.
+
+```
+$ npm run start -- --help
+
+index.ts [command]
+
+Commands:
+  index.ts overview  list stacks, contained resources, and if @guardian/cdk managed
+
+Options:
+  --help          Show help                                            [boolean]
+  --version       Show version number                                  [boolean]
+  --prefer-cache  use local cache and fail is not found                [boolean]
```

Comment:
I don't think it will fail if not found ATM due to the [`existsSync` check](https://github.com/guardian/cloudformation-information/blob/8a85ac704ab1e38ef1615d61a59a904aec6f4afa/src/cloudformation.ts#L113).